Libs/Particles/ParticleSystem.h

```diff
@@ -9,7 +9,7 @@ class ParticleSystem
 public:
   ParticleSystem();
 
-  bool LoadParticles(const std::vector<std::string> &paths);
+  void LoadParticles(const std::vector<std::string> &paths);
```

Comment:
Is ParticleSystem like Image in the sense that the only valid instantiation has loaded a file? If so, why not emulate Image and put the argument of file names in the constructor, making a default constructor private?

Reply:
Done. I've made a new PR, closing this
https://github.com/SCIInstitute/ShapeWorks/pull/522